scripts/publish-documentation/README.md

```diff
@@ -136,20 +152,27 @@ After running the example, the following happens:
     │
     ```
 
-#### Edge cases
+### Edge cases
 
 - If any of the passed tags already has Dokka documentation published to the `gh-pages` branch, the 
   script tries to overwrite it if there are changes in a tool output. The presence of changes 
   depends on how much the Dokka configuration has changed since the last publication. If an output 
   matches the published one, a commit is not made.
 
+
 - If no tags are marked 'primary', they all are considered 'secondary' and processed accordingly.
 
+
 - If multiple tags are marked 'primary', then each of these tags is published as 'primary' one by one. 
   That means the last in the list marked as 'primary' overwrites all others as if they were published
   as 'secondary'.
 
 ### OS details
 
 The script was developed under and for the macOS. It should not have problems working on a Linux 
-distribution. However, it was not meant and tested to do so.
\ No newline at end of file
+distribution. However, it was not meant and tested to do so.
+
+[jenv]: https://github.com/jenv/jenv#12-adding-your-java-environment
+[jenv-add]: https://github.com/jenv/jenv#12-adding-your-java-environment
```

Comment:
Isn't this link the same as the previous one? Please review.

Reply:
It is, indeed. It should be just https://github.com/jenv/jenv.